toolbox/providers/gcp/gcp-advanced.md

```diff
@@ -1,40 +1,46 @@
-# # Nauta on Google Cloud Platform - Advanced options
+# # Nauta on Google Cloud* Platform: Advanced Options
 
 This document contains information referring to [Nauta on Google Cloud Platform - Getting Started](gcp.md).
 
-## Overall cluster architecture
+## Overall Cluster Architecture
+
 GCP Nauta Kubernetes instance consists of several parts:
-- gateway - bastion host to access to cluster
-- test node (optional) - bastion host with test software to do continuous tests
-- nfs server - internal server with nfs based resources
-- gke cluster - kubernetes cluster.
 
-Please refer to terraform definition files (in directory terraform/templates) to see detailed configuration (networking,
+- **gateway:** Bastion host to access to cluster
+- **test node: (optional)** Bastion host with test software to do continuous tests
+- **nfs server:** Internal server with nfs based resources
+- **gke cluster:** - Kubernetes cluster
+
+Refer to terraform definition files (in directory terraform/templates) to see detailed configuration (networking,
 resources, etc.)
 
-## Prerequisities
+## Prerequisites
 
 ### Workstation
-To install Nauta on GCP workstation (Linux or Mac based) with access to the internet is required. Installation repository (nauta)
-has to be downloaded or cloned from github repository.
+
+To install Nauta on GCP workstation (Linux or Mac-based) with access to the internet is required. Installation repository (nauta) has to be downloaded or cloned from github repository.
 
 ***All operations described below should be made on workstation. All paths are connected with downloaded or cloned repository*** 
```

Comment:
Should there be a reference on how to clone Nauta; and info on where to download the tar file?